frappe/website/doctype/help_article/templates/help_article.html

```diff
@@ -19,14 +19,45 @@ <h6 class='text-muted'>By {{ author }} on {{ frappe.format_date(creation) }}</h6
 	<p><br><a href="/{{ category.route }}" class='text-muted small'>
 		{{ _("More articles on {0}").format(category.name) }}</a></p>
 </article>
+<div class="help-article-feedback mb-4">
+	<hr />
+	<div class="feedback-view ">
+		<span class="text-muted small" style="margin-right: 15px;">{{ _("Was this helpful?") }}</span>
+		<button class="feedback btn btn-outline-primary btn-sm" data-value="Yes" style="width: 50px;">{{ _("Yes") }}</button>
+		<button class="feedback btn btn-outline-primary btn-sm" data-value="No" style="width: 50px;">{{ _("No") }}</button>
+	</div>
+</div>
 <div class="help-article-comments">
 	<hr>
 	<h4>Comments</h4>
 	{% include 'templates/includes/comments/comments.html' %}
 </div>
 <script>
 frappe.ready(function() {
-    frappe.set_search_path("/kb");
+	frappe.set_search_path("/kb");
+	$(".feedback").click(function() {
+		let args = {
+			article: "{{ reference_name or name }}",
+			helpful: this.getAttribute("data-value"),
+		}
+
+		let disable_button = function(btn) {
+			btn.classList.remove("btn-outline-primary");
+			btn.classList.add("btn-light");
+			btn.disabled = true
+		}
+
+		frappe.call({
+			btn: this,
+			method: "frappe.website.doctype.help_article.help_article.add_feedback",
+			args: args,
+			callback: function(r) {
+				disable_button($(".feedback")[0]);
+				disable_button($(".feedback")[1]);
+				frappe.msgprint(__("Thank you for your feedback!"));
```

Comment:
Instead of message print, we can show the thank you there in the same section itself